src/types.go

```diff
@@ -86,10 +90,12 @@ type Tournament struct {
 
 // UserTournamentStats object in datastore represents an user's performance in a particular tournament
 type UserTournamentStats struct {
-	UserID       int64
-	TournamentID int64
-	FFAWins      int
-	Wins         int
-	Losses       int
-	Rating       float64
+	UserID         int64
+	TournamentID   int64
+	FFAWins        int
+	Wins           int
+	Losses         int
+	Rating         float64
+        TrueSkillMu    float64
+        TrueSkillSigma float64
```

Comment:
I think you can deploy to a staging version and check whether the current data still works.

Reply:
https://cloud.google.com/appengine/docs/flexible/python/testing-and-deploying-your-app
gcloud app deploy --project PROJECT_ID --version VERSION_ID --no-promote

Reply:
Tested in https://cookieben-dev-dot-elo-site-201802.appspot.com/tournament/TTA.